tests/integrationv2/common.py

```diff
@@ -165,40 +165,40 @@ class Ciphers(object):
-    ECDHE_ECDSA_AES128_SHA256 = Cipher("ECDHE_ECDSA_AES128_SHA256", Protocols.TLS12, True, True)
-    ECDHE_ECDSA_AES256_SHA384 = Cipher("ECDHE_ECDSA_AES256_SHA384", Protocols.TLS12, True, True)
-    ECDHE_ECDSA_AES128_GCM_SHA256 = Cipher("ECDHE_ECDSA_AES128_GCM_SHA256", Protocols.TLS12, True, True)
-    ECDHE_ECDSA_AES256_GCM_SHA384 = Cipher("ECDHE_ECDSA_AES256_GCM_SHA384", Protocols.TLS12, True, True)
-    ECDHE_ECDSA_CHACHA20_POLY1305 = Cipher("ECDHE_ECDSA_CHACHA20_POLY1305", Protocols.TLS12, True, False)
-
-    ECDHE_RSA_DES_CBC3_SHA = Cipher("ECDHE_RSA_DES_CBC3_SHA", Protocols.SSLv3, False, False)
-    ECDHE_RSA_AES128_SHA = Cipher("ECDHE_RSA_AES128_SHA", Protocols.SSLv3, True, False)
-    ECDHE_RSA_AES256_SHA = Cipher("ECDHE_RSA_AES256_SHA", Protocols.SSLv3, True, False)
-    ECDHE_RSA_RC4_SHA = Cipher("ECDHE_RSA_RC4_SHA", Protocols.SSLv3, False, False)
-    ECDHE_RSA_AES128_SHA256 = Cipher("ECDHE_RSA_AES128_SHA256", Protocols.TLS12, True, True)
-    ECDHE_RSA_AES256_SHA384 = Cipher("ECDHE_RSA_AES256_SHA384", Protocols.TLS12, True, True)
-    ECDHE_RSA_AES128_GCM_SHA256 = Cipher("ECDHE_RSA_AES128_GCM_SHA256", Protocols.TLS12, True, True)
-    ECDHE_RSA_AES256_GCM_SHA384 = Cipher("ECDHE_RSA_AES256_GCM_SHA384", Protocols.TLS12, True, True)
-    ECDHE_RSA_CHACHA20_POLY1305 = Cipher("ECDHE_RSA_CHACHA20_POLY1305", Protocols.TLS12, True, False)
-    CHACHA20_POLY1305_SHA256 = Cipher("CHACHA20_POLY1305_SHA256", Protocols.TLS13, True, False)
+    DHE_RSA_DES_CBC3_SHA = Cipher("DHE-RSA-DES-CBC3-SHA", Protocols.SSLv3, False, False)
+    DHE_RSA_AES128_SHA = Cipher("DHE-RSA-AES128-SHA", Protocols.SSLv3, True, False, TEST_CERT_DIRECTORY + 'dhparams_2048.pem')
+    DHE_RSA_AES256_SHA = Cipher("DHE-RSA-AES256-SHA", Protocols.SSLv3, True, False, TEST_CERT_DIRECTORY + 'dhparams_2048.pem')
+    DHE_RSA_AES128_SHA256 = Cipher("DHE-RSA-AES128-SHA256", Protocols.TLS12, True, True, TEST_CERT_DIRECTORY + 'dhparams_2048.pem')
+    DHE_RSA_AES256_SHA256 = Cipher("DHE-RSA-AES256-SHA256", Protocols.TLS12, True, True, TEST_CERT_DIRECTORY + 'dhparams_2048.pem')
+    DHE_RSA_AES128_GCM_SHA256 = Cipher("DHE-RSA-AES128-GCM-SHA256", Protocols.TLS12, True, True, TEST_CERT_DIRECTORY + 'dhparams_2048.pem')
+    DHE_RSA_AES256_GCM_SHA384 = Cipher("DHE-RSA-AES256-GCM-SHA384", Protocols.TLS12, True, True, TEST_CERT_DIRECTORY + 'dhparams_2048.pem')
+    DHE_RSA_CHACHA20_POLY1305 = Cipher("DHE-RSA-CHACHA20-POLY1305", Protocols.TLS12, True, False, TEST_CERT_DIRECTORY + 'dhparams_2048.pem')
+
+    AES128_SHA = Cipher("AES128-SHA", Protocols.SSLv3, True, True)
+    AES256_SHA = Cipher("AES256-SHA", Protocols.SSLv3, True, True)
+    AES128_SHA256 = Cipher("AES128-SHA256", Protocols.TLS12, True, True)
+    AES256_SHA256 = Cipher("AES256-SHA256", Protocols.TLS12, True, True)
+    AES128_GCM_SHA256 = Cipher("TLS_AES_128_GCM_SHA256", Protocols.TLS13, True, True)
+    AES256_GCM_SHA384 = Cipher("TLS_AES_256_GCM_SHA384", Protocols.TLS13, True, True)
```

Comment:
nit: Is there a reason for underscore here when the others have been converted to hyphenated?

Reply:
Those ciphers are TLS13 specific, and that is how OpenSSL wants them (`openssl ciphers`). It made more sense to match OpenSSL's format here then try to have our own format and convert to the provider's format at runtime.